### PR TITLE
adding BAL-A

### PR DESCRIPTION
**Pull request by godsflaw**
# Description

Spell Description:

Adds BAL-A to kovan

COB Team Name or Author(s): @hexonaut @godsflaw 

# Contribution Checklist

- [x] Code approved
- [x] Tests approved
- [x] CI Tests pass

# Checklist

- [x] Every contract variable/method declared as public/external private/internal
- [x] Verify expiration (`4 days + 2 hours` monthly and `30 days` for the rest)
- [x] Verify hash in the description matches [here](https://emn178.github.io/online-tools/keccak_256.html)
- [x] Validate all addresses used are in Kovan changelog
- [x] Deploy spell to kovan `SOLC_FLAGS="--optimize --optimize-runs=1" dapp --use solc:0.5.12 --network kovan build --extract && dapp create DssSpell --verify --network kovan --gas=XXX --gas-price="$(seth --to-wei YYY "gwei")"`
- [x] Ensure contract is verified on `kovan` etherscan
- [x] Change test to use kovan spell address and deploy timestamp
- [x] Keep `Kovan-DssSpell.sol` and `Kovan-DssSpell.t.sol` the same, but make a copy in `archive`
- [x] `squash and merge` this PR

### CODE REVIEW

**Review by hexonaut — APPROVED** [review]
LGTM

File: template/SpellAction.t.sol [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        // add to the end of the list of collateral tests
+        // change the values as appropriate
+        afterSpell.collaterals["TOKEN-LETTER"] = CollateralValues({
+            line:         4 * MILLION,
+            dust:         100,
+            pct:          500,
+            chop:         1300,
+            dunk:         500,
+            mat:          17500,
+            beg:          10300,
```
Comment (brianmcmichael) [inline template/SpellAction.t.sol:17]: Should be `300` now

**Comment by godsflaw** [comment]
```
Running 2 tests for src/Kovan-DssSpell.t.sol:DssSpellTest
[PASS] testSpellIsCast_BAL_INTEGRATION() (gas: 2355104)
[PASS] testSpellIsCast() (gas: 4307143)
```

**Review by brianmcmichael — APPROVED** [review]

File: template/SpellAction.t.sol (hunk 1 above)
Comment (godsflaw) [reply template/SpellAction.t.sol]: fixes

**Comment by godsflaw** [comment]
```
Running 2 tests for src/Kovan-DssSpell.t.sol:DssSpellTest
[PASS] testSpellIsCast_BAL_INTEGRATION() (gas: 2355104)
[PASS] testSpellIsCast() (gas: 4307130)
```

**Pull request merged**